package.json

```diff
@@ -2,6 +2,9 @@
   "name": "go-ipfs-dep",
   "version": "0.4.4",
   "description": "Install the latest go-ipfs binary",
+  "go-ipfs": {
+    "version": "v0.4.4"
+  },
```

Comment:
Needs engines set to something like this: https://github.com/ipfs/js-ipfs-api/blob/master/package.json#L51-L54  as it is now using things like arrow functions, which are not available earlier